app/views/root/local_transaction.html.erb

```diff
@@ -58,3 +58,17 @@
     </div>
   </section>
 <% end %>
+
+<% if @postcode_error %>
+  <script type="text/javascript">
+    //<![CDATA[
```

Comment:
I'll remove the `CDATA`. As for not having an inline script, suggestions welcome.